controller/api/urls.py

```diff
@@ -1,41 +1,36 @@
 from __future__ import unicode_literals
 
 from django.conf import settings
-from django.conf.urls import include
 from django.conf.urls import patterns
 from django.conf.urls import url
 
-from api import routers
 from api import views
 
 
-router = routers.ApiRouter()
-
 # Add the generated REST URLs and login/logout endpoint
 urlpatterns = patterns(
     '',
-    url(r'^', include(router.urls)),
```

Comment:
If possible, we should leave them in. I think the browseable API pages had their own login page so they were actually functional, since I didn't see how to use token auth with them. But I hadn't checked in a while.